src/c_api.cpp

```diff
@@ -118,6 +118,7 @@ class Booster {
     if (config_.num_threads > 0) {
       omp_set_num_threads(config_.num_threads);
     }
+
```

Comment:
since your only changes in this file are whitespace, can you please revert them? That will make git blame cleaner and remove one file that has to be reviewed

Reply:
Yes.  In the process of doing so right now.

Reply:
thanks!

Reply:
Some introduced compiler warnings in master.  I think these should be fixed.

```
src/c_api.cpp:1967:5: warning: unused parameter 'num_iteration' [-Wunused-parameter]
 int LGBM_BoosterPredictForCSRSingleRowFast(FastConfigHandle fastConfig_handle,
     ^
src/c_api.cpp:2104:5: warning: unused parameter 'num_iteration' [-Wunused-parameter]
 int LGBM_BoosterPredictForMatSingleRowFast(FastConfigHandle fastConfig_handle,
     ^
```

Reply:
> Someone introduced compiler warnings in master.

They will be fixed in #3268.

Reply:
ooo thanks! I just saw these last night with `gcc` too